packages/beacon-node/src/api/impl/beacon/pool/index.ts

```diff
@@ -224,7 +234,7 @@ export function getBeaconPoolApi({
       );
 
       if (errors.length > 1) {
-        throw Error("Multiple errors on publishAggregateAndProofs\n" + errors.map((e) => e.message).join("\n"));
+        throw Error("Multiple errors on submitPoolSyncCommitteeSignatures\n" + errors.map((e) => e.message).join("\n"));
```

Comment:
This just fixes a typo but it is worth highlighting that we don't need to handle already known errors in submitPoolSyncCommitteeSignatures as the api only verifies the signature
https://github.com/ChainSafe/lodestar/blob/d7ca290d173dfa82b2977f3e94df6e5e8d12ef3e/packages/beacon-node/src/api/impl/beacon/pool/index.ts#L184-L186